cmd/agent-js-worker/main.go

```diff
@@ -153,6 +153,7 @@ func getAriesHandlers(ctx *context.Provider, r controllercmd.MessageHandler,
 			EDVBatchEndpointExtensionEnabled: true,
 			WebKMSGNAPSigner:                 headerFunc,
 			EDVGNAPSigner:                    headerFunc,
+			ValidateDataModel:                true,
```

Comment:
this should be configurable, you should read it from wallet config. Shouldn't be hardcoded to `True`.

Reply:
currently, it's only a vc wallet config parameter, we should probably extend afgo start up options to add it there and use it as an option

Reply:
done